apps/music_proxy.cpp

```diff
@@ -39,13 +39,17 @@
 
 namespace po = boost::program_options;
 
+namespace
+{
+const double defaultMusicTimestep = 0.0001;
+const brion::URI brionMonsteerSpikesPluginURI( "monsteer://" );
```

Comment:
Yes. I thought about, it but I don't know where we could define that identifier in a common place? How is this done for other plugins selection?